src/PubSub/Drivers/RedisClient.php

```diff
@@ -60,6 +68,7 @@ class RedisClient extends LocalClient
     public function __construct()
     {
         $this->serverId = Str::uuid()->toString();
+        $this->redis = Cache::getRedis();
```

Comment:
Would it not be better to use dependency injection here?

Reply:
It also seems like a really bad idea to use `\Illuminate\Redis\RedisManager` because that is a synchronous API. You really should use the `ReactPHP` redis client, i.e. `publishClient`, for this.

Reply:
I have used it because it was easier to test for anything, except for the pub/sub protocol. Gonna revert this.